src/home_robot_hw/home_robot_hw/ros/camera.py

```diff
@@ -47,7 +47,7 @@ def __init__(
         self.buffer_size = buffer_size
         if self.buffer_size is not None:
             # create buffer
-            self._buffer = deque()
+            self._buffer: deque = deque()
```

Comment:
I dont understand this change

Reply:
This is just typing. Mypy was complaining about it, but that was before I gave up on enabling it for now so we can remove this change.